tools/tcat_ble_client/cli/base_commands.py

```diff
@@ -379,3 +384,14 @@ async def execute_default(self, args, context):
         else:
             print('Secure channel not established.')
             await ble_stream.disconnect()
+        return CommandResultNone()
+
+
+class DisconnectCommand(Command):
+
+    def get_help_string(self) -> str:
+        return 'Disconnect client from TCAT device'
+
+    async def execute_default(self, args, context):
+        await context['ble_sstream'].close()
```

Comment:
should we check if context exists here as well?

Reply:
great catch thanks.